loggy/lib/src/filters/whitelist_filter.dart

```diff
@@ -0,0 +1,14 @@
+part of loggy;
+
+/// White list filter will log only [Type] of loggy that is passed.
+/// You should pass only [Type] that is from [LoggyType] otherwise no effect will take place.
+class WhitelistFilter extends LoggyFilter {
+  const WhitelistFilter(this._types);
+
+  final List<Type> _types;
+
+  @override
+  bool shouldLog(LogLevel level, Type type) {
+    return _types.isEmpty || _types.contains(type);
```

Comment:
I'm a bit confused here.
The documentation is not in sync with the behavior (I think you should change the behavior).

If I understand this correctly, the `WhitelistFilter` only shows logs from `LoggyType`-s that are passed.
So if I create a `WhitelistFilter([])` nothing should be logged. 

In the example below:
The log is: `👻 08:05:46.351346 INFO     Example - abc` 
I would expect the log to be empty

```dart
void main() {
  Loggy.initLoggy(
    logPrinter: const PrettyPrinter(),
    logOptions: const LogOptions(
      LogLevel.all,
    ),
    filters: [
      WhitelistFilter([]),
    ],
  );

  MyExample();
}

class MyExample with ExampleLoggy {
  MyExample(){
    loggy.info("abc");
  }
}
```

Reply:
Oh wow, good catch. Anyway I updated behaviour to make sense now (For whitelist and blacklist) and added tests for filters with that cases :)